app/models/sales_log.rb

```diff
@@ -318,6 +318,7 @@ def reset_created_by!
     return unless updated_by&.support?
     return if owning_organisation.blank? || created_by.blank?
     return if created_by&.organisation == owning_organisation
+    return if created_by&.organisation == owning_organisation.absorbing_organisation
```

Comment:
we'll need to add managing to this too once the sales managing org behaviour is in, right?